server/graphql/v1/CollectiveInterface.js

```diff
@@ -744,7 +745,12 @@ const CollectiveFields = () => {
     },
     createdByUser: {
       type: UserType,
-      resolve(collective) {
+      resolve(collective, args, req) {
+        if (
+          collective.isAnonymous &&
+          (!req.remoteUser || !req.remoteUser.isAdmin(collective.inTheContextOfCollectiveId))
+        )
+          return {};
```

Comment:
Otherwise it breaks down the graph when you try to return `user.firstName` for example.
Otherwise you need to always test that `user` passed to the resolver is not null.